src/ape/types/__init__.py

```diff
@@ -31,6 +31,11 @@
 as well as a block number or hash (HexBytes).
 """
 
+ContractCode = Union[str, bytes, HexBytes]
+"""
+An ID that represents contract code, which can be represented in string, bytes, or HexBytes.
```

Comment:
It is not really an ID here

```suggestion
A type that represents contract code, which can be represented in string, bytes, or HexBytes.
```